src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/personalrelationships/ContactPersonDpsApiService.kt

```diff
@@ -106,8 +106,8 @@ class ContactPersonDpsApiService(@Qualifier("personalRelationshipsApiWebClient")
     .retrieve()
     .awaitBodyWithRetry(backoffSpec)
 
-  suspend fun getLinkedPrisonerContacts(contactId: Long): List<LinkedPrisonerDetails> = webClient.get()
-    .uri("/contact/{contactId}/linked-prisoners", contactId)
+  suspend fun getLinkedPrisonerContacts(contactId: Long): PagedModelLinkedPrisonerDetails = webClient.get()
+    .uri("/contact/{contactId}/linked-prisoners?page=0&size=2000", contactId)
     .retrieve()
```

Comment:
missed that did was switched to paging - 2000 sounds arbitrary - there are some staff person records with lots of contacts

Reply:
Highest is 6868 - so maybe I need to paging correctly